inference-engine/ie_bridges/rust/crates/openvino-sys/upstream

```diff
@@ -0,0 +1 @@
+../../../../..
```

Comment:
This creates a symlink cycle (upstream/inference-engine/ie_bridge/rust/crates/openvino-sys/upstream`. This may not work on Windows.

Reply:
The other option here was to create a separate repository for the Rust code and bring in OpenVINO as a submodule. But this then creates distance between the Rust code and the C API which means additional work to keep them in sync. It seemed easier to keep things working if the "bridges" are in the same repository and I will probably always publish crates from Linux, so this shouldn't be an issue (from what I observed, all the symlinks seemed resolved once Cargo packaged up the crate--right?).